platform/web/js/libs/library_godot_os.js

```diff
@@ -291,6 +291,27 @@ const GodotOS = {
 		});
 	},
 
+	godot_js_os_has_feature__sig: 'ii',
+	godot_js_os_has_feature: function (p_ftr) {
+		const ftr = GodotRuntime.parseString(p_ftr);
+		if (ftr === 'web_macos') {
+			return (navigator.appVersion.indexOf('Mac') !== -1) ? 1 : 0;
```

Comment:
`navigator.appVersion` is deprecated: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/appVersion
We should probably use [`navigator.userAgent`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/userAgent) instead like done in [`GodotInputGamepads.get_guid`](https://github.com/godotengine/godot/blob/master/platform/web/js/libs/library_godot_input.js#L122).

Reply:
Changed it to `navigator.userAgent`.